<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buscarMiembro</class>
 <widget class="QDialog" name="buscarMiembro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar Miembro</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Buscar Miembro</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Buscar por:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Optioncombo">
        <item>
         <property name="text">
          <string>Todo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nombre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Apellido Paterno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Apellido Materno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Grado de Estudio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ocupacion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Estado Civil</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Numero</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputTxt"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="resultadotableWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>180</y>
     <width>931</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
